pkg/materialize/source_load_generator.go

```diff
@@ -9,7 +9,6 @@ import (
 
 type CounterOptions struct {
 	TickInterval   string
-	ScaleFactor    float64
```

Comment:
We appear to be marking the field as deprecated, but are actually removing support for it?

Reply:
It was never supported. The value was just ignored, now the load generator is more strict and complains if you pass nonsensical arguments.

Reply:
Will the command still succeed if a scale factor is provided, or will it error out?

Reply:
Error.

Reply:
Got it. In that case I don't think we should be marking it as deprecated - we should remove it altogether. Deprecation generally implies that a feature still works, but will stop working soon.

Reply:
Yep, the idea behind the deprecation warning at first is that I think this will officially land in v0.93.x so in a couple of weeks. So once it's officially there we'll go ahead and also remove the attribute from the schema.

Reply:
Got it! This makes sense then, especially since the field does nothing when this provider is aimed at current-stable Mz.